Sources/Frontend/Formatters/JsonFormatter.swift

```diff
@@ -34,6 +35,7 @@ final class JsonFormatter: OutputFormatter {
                 for ref in references {
                     let object: [AnyHashable: Any] = [
                         "kind": ref.kind.rawValue,
+                        "modules": Array(result.declaration.location.file.modules),
```

Comment:
This part is redundant and can be removed, as it's the same as the `modules` from the parent.

Reply:
Thanks, I fixed it in https://github.com/peripheryapp/periphery/pull/519/commits/c3a12bc560c53c39fb0e837d09701b1ef5e91910